### PR TITLE
Add v0.16.1.0 release notes

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -52,553 +51,52 @@ so downgrading is not recommended.
...
-
-Build system
-------------
-The minimum version of the GCC compiler required to compile Dash Core is now 4.8.
-The minimum version of Qt is now 5.5.1. Some packages in `depends/` as well as
-`secp256k1` and `leveldb` subtrees were updated to newer versions.
+Fixed an issue when wallet would refuse to load if no valid GUI theme was found in settings.
+This was happenning for some users who had pre-v0.14 wallets installed previousely.
+Few other GUI issues like fonts acting strange sometimes, window size not updating correctly,
+a couple of crashes which could occur when a node is compiled or started with wallets disabled etc.
```
Comment (xdustinface) [inline doc/release-notes.md:64]:
```suggestion
a couple of crashes which could occur when a node is compiled or started with wallets disabled
```

File: doc/release-notes.md [hunk 2]
```diff
@@ -52,553 +51,52 @@ so downgrading is not recommended.
...
-be mixed for another round. This rule is then executed again on the new inputs.
-This results in an exponential decay where if you mix a set
-of inputs, half of those inputs will be mixed for N rounds, 1/4 will be mixed N+1,
-1/8 will be mixed N+2, etc. Current implementation caps it at N+3 which results
-in mixing an average of N+0.875 rounds and sounds like a reasonable compromise
-given the privacy gains.
+Network changes
+---------------
+InstantSend locks were not relayed correctly when another node was requesting updates via `mempool`
+p2p message. Some other internal optimizations were made to the way object requests are handled.
```
Comment (xdustinface) [inline doc/release-notes.md:57]: Probably not very important to mention but weren't the "Some other internal.." rather bugfixes than optimizations? (of course, bugfixes also optimize things 😆)

**Review by xdustinface — CHANGES_REQUESTED** [review]
What about either the suggestions in the comments or maybe just bc28d901efa94444b87d83407b46a31cf9036d91, thats what i came up with while/after adding the comments :)

File: doc/release-notes.md [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
 
 Release is now available from:
 
   <https://www.dash.org/downloads/#wallets>
 
-This is a new major version release, bringing new features, various bugfixes
-and other improvements.
+This is a new minor version release, bringing various bugfixes and improvements.
 
 This release is mandatory for all nodes.
```
Comment (thephez) [inline doc/release-notes.md:10]: Is this accurate?

File: doc/release-notes.md [hunk 4]
```diff
@@ -52,553 +51,52 @@ so downgrading is not recommended.
...
-at this point, a salted hash of each input is used to pick ~50% of inputs to
-be mixed for another round. This rule is then executed again on the new inputs.
-This results in an exponential decay where if you mix a set
-of inputs, half of those inputs will be mixed for N rounds, 1/4 will be mixed N+1,
-1/8 will be mixed N+2, etc. Current implementation caps it at N+3 which results
-in mixing an average of N+0.875 rounds and sounds like a reasonable compromise
-given the privacy gains.
+Network changes
+---------------
+InstantSend locks were not relayed correctly when another node was requesting updates via `mempool`
```
Comment (thephez) [inline doc/release-notes.md:56]:
```suggestion
InstantSend locks were not relayed correctly when another node was requesting updates via the `mempool`
```

File: doc/release-notes.md [hunk 5]
```diff
@@ -52,553 +51,52 @@ so downgrading is not recommended.
...
-and `SPORK_20_INSTANTSEND_LLMQ_BASED` which were previously deprecated in v0.15
-are completely removed now. `SPORK_6_NEW_SIGS` was never activated on mainnet
-and was also removed in this version.
-
-Build system
-------------
-The minimum version of the GCC compiler required to compile Dash Core is now 4.8.
-The minimum version of Qt is now 5.5.1. Some packages in `depends/` as well as
-`secp256k1` and `leveldb` subtrees were updated to newer versions.
+Fixed an issue when wallet would refuse to load if no valid GUI theme was found in settings.
```
Comment (thephez) [inline doc/release-notes.md:61]:
```suggestion
Fixed an issue where the wallet would refuse to load if no valid GUI theme was found in settings.
```

File: doc/release-notes.md [hunk 6]
```diff
@@ -52,553 +51,52 @@ so downgrading is not recommended.
...
+This was happenning for some users who had pre-v0.14 wallets installed previousely.
+Few other GUI issues like fonts acting strange sometimes, window size not updating correctly,
```
[the remark this answers is not in the document]
Comment (thephez) [reply doc/release-notes.md:63]: Maybe "... like inconsistently scaled font sizes, ..."

**Review by thephez — COMMENTED** [review]
A few small comments

**Review by xdustinface — APPROVED** [review]
utACK

File: doc/release-notes.md [hunk 7]
```diff
@@ -52,553 +49,56 @@ so downgrading is not recommended.
...
-are completely removed now. `SPORK_6_NEW_SIGS` was never activated on mainnet
-and was also removed in this version.
-
-Build system
-------------
-The minimum version of the GCC compiler required to compile Dash Core is now 4.8.
-The minimum version of Qt is now 5.5.1. Some packages in `depends/` as well as
-`secp256k1` and `leveldb` subtrees were updated to newer versions.
+Fixes for the following GUI related issues
+- The wallet did refuse to load if no valid GUI theme was found in settings.
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:60]:
```suggestion
- The wallet crashed if no valid GUI theme was found in settings.
```

File: doc/release-notes.md [hunk 8]
```diff
@@ -52,553 +49,56 @@ so downgrading is not recommended.
...
+This happened for users upgrading from versions prior to v0.14.
+- There were randomly occurring font size scaling issues like abnormal font sizes.
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:62]:
```suggestion
- There were randomly occurring font size scaling issues.
```

File: doc/release-notes.md [hunk 9]
```diff
@@ -52,553 +49,56 @@ so downgrading is not recommended.
...
+- Open and close the settings while the application was in fullscreen/maximised
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:63]:
```suggestion
- Opening or closing the settings while the application was in full-screen/maximized
```

File: doc/release-notes.md [hunk 10]
```diff
@@ -52,553 +49,56 @@ so downgrading is not recommended.
...
+window mode lead to fragmented GUI layouts.
+- There was an unexpected checkmark in the "Encrypt wallet" menu item on linux/windows
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:65]:
```suggestion
- There was an unexpected checkmark in the "Encrypt wallet" menu item on Linux and Windows
```

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]

**Review by xdustinface — APPROVED** [review]
re-utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Review by thephez — APPROVED** [review]
ACK

**Pull request merged**